commit 4c18467f4bd17e70f940203d6e6d8de4e01bd762 (refs/changes/25/625/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2016-10-07T23:04:26+00:00 (3 years ago)
    
    INTRO-720 The instrumenter shouldn't run in RT
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/smf/method/canodesvc b/smf/method/canodesvc
index cb090b6..01d0634 100755
--- a/smf/method/canodesvc
+++ b/smf/method/canodesvc
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2016, Joyent, Inc.
 #
 
 #
@@ -91,11 +91,11 @@ function load_logfiles
 	ns_rqlogfile="$ns_logbase/$ns_name/requests-$(date +%F.%T).$$"
 }
 
-function run_realtime
+function run_fx
 {
 	local pid=$1
 
-	priocntl -s -c RT -i pid $pid
+	priocntl -s -c FX -m 60 -p 60 -i pid $pid
 	[[ $? != 0 ]] && echo "Failed to move process into the RT class"
 }
 
@@ -170,5 +170,5 @@ env | sort | egrep '^(AMQP_|CNAPI_|VMAPI_|NODE_PATH)' | grep -v PASSWORD
 smf_clear_env
 $ns_node $ns_exec $ns_args &
 ns_pid=$!
-[[ $ns_rt == "true" ]] && run_realtime $ns_pid
+[[ $ns_rt == "true" ]] && run_fx $ns_pid
 exit $SMF_EXIT_OK
